hardhat.config.js

```diff
@@ -35,11 +35,18 @@ const optimizerSettings = {
     },
 };
 
+const outputSelectionSettings = {
+    "*": {
+        "*": ["storageLayout"],
+    },
+};
+
 const defaultSettings = {
     version: '0.8.23',
     settings: {
         evmVersion: process.env.EVM_VERSION || 'london',
         optimizer: optimizerSettings,
+        outputSelection: outputSelectionSettings,
```

Comment:
coverage action is failing, maybe because this should only be set if the storage layout env is set?